mobile/ios/device/ios-core.ts

```diff
@@ -749,23 +751,27 @@ export class WinSocket implements Mobile.IiOSDeviceSocket {
 
 	public close(): void {
 		this.winSocketLibrary.closesocket(this.service);
+		if (this.sysLogChildProcess) {
+			this.sysLogChildProcess.kill();
```

Comment:
In addition, I may be wrong here, but I'm not sure that calling `.kill` will detach all event handlers from the child process. Assigning it to `null` could help because if it gets garbage collected the event handlers will disappear with it, if not however it could cause a potential memory leak

@rosen-vladimirov any thoughts on this?